test/verify/check-setroubleshoot

```diff
@@ -143,6 +149,18 @@ class TestSelinux(MachineCase):
         b.click(row_selector + " button.pficon-delete")
         b.wait_not_present(row_selector)
 
+        if "distropkg" in m.image: # Introduced in #11789
```

Comment:
Please enumerate rhel-8-0-distropkg explicitly here. We'll have e. g. 8-{1,2}-distropkg at some point and don't want to skip all of them.